src/services/api/categories.js

```diff
@@ -4,6 +4,8 @@ import { SITES, CATEGORIES } from 'common/constants'
 const categories = {
   get: async ({ sitename, id }) =>
     apiBase().get(`${SITES}/${sitename}/${CATEGORIES}/${id}`).json(),
+  getAll: async ({ sitename }) =>
```

Comment:
Just wondering, can we want to make the `id` conditional in the above `get` method ?
So then it looks more consistent with the http verbs and APIs, like if we pass an id, we get the specific object, if not, we get the whole list ?

Reply:
Good question. My preference would be to stick with separate methods in the FE for clarity around what is required for each desired response. The full lists also sometimes require other params e.g. pageSize.